readme.md

```diff
@@ -268,6 +269,9 @@ If a particular file sends an event, then it would be `'myFile.functionWhereEven
 
 _IMPORTANT! `eventHubEvents: {}` can only be used in T1Views. If you are using a vanilla Backbone view, the receiver view must manually subscribe in order to listen for a published event (e.g. `T1.EventHub.subscribe('eventName':'callBack')`_
 
+*Other important notes about eventhub:*  event hub events **must** be unregistered when views are unloaded or they'll be called from other pages.
```

Comment:
Is this true of T1Views or just the backbone views? the `unload` function in T1View calls `unbind`, which in turn calls `unbindDataEvents` and `unbindEventHubEvents`